CMakeLists.txt

```diff
@@ -144,12 +144,16 @@ target_include_directories(mbed-core
 add_library(mbed-device_key INTERFACE)
 add_library(mbed-rtos INTERFACE)
 
+# Include targets/ first, because any post-build hook needs to be defined
```

Comment:
In the commit message for commit "Rework post-build to support multiple executables", we say "guard it with a macro to **checks** if", that should be "guard it with a macro to **check** if"

Reply:
Good catch. I probably forgot to change it when rephrasing the sentence. 😅